nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/test/java/org/nuxeo/ecm/platform/comment/TestBridgeFromPropertyToTreeCommentManager.java

```diff
@@ -134,4 +135,19 @@ public void shouldGetTopLevelCommentAncestorAsTree() {
         assertEquals(getCommentedDocument().getRef(), topLevelCommentAncestor);
     }
 
+    @Override
+    protected DocumentRef getCommentedDocRef(CoreSession session, DocumentModel commentDocModel, boolean reply) {
+        DocumentRef parentRef = commentDocModel.getParentRef();
+        // Comment is under property model (a regular comment or a reply)
+        if (session.getDocument(parentRef).getType().equals("HiddenFolder")) {
+            return new IdRef((String) commentDocModel.getPropertyValue(COMMENT_PARENT_ID));
+        }
+
+        // Comment as Tree
+        if (reply) {
+            return commentDocModel.getParentRef();
+        }
+        return commentManager.getTopLevelCommentAncestor(session, commentDocModel.getRef());
+    }
+
```

Comment:
Why not using the one from services?

Reply:
this is because if test the behaviour of the `getCommentedDocRef` of the the `CommentManager` for example this test: `org.nuxeo.ecm.platform.comment.AbstractTestCommentManager#shouldGetCommentedDocModel`